src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/PersonResourceTest.kt

```diff
@@ -93,6 +93,18 @@ class PersonResourceTest : ResourceTest() {
         .expectBody()
         .json("[]")
     }
+
+    @Test
+    fun `returns multiple addresses for addresses with single and multiple phones`() {
+      webTestClient.get().uri("/api/persons/-8/addresses")
+        .headers(setClientAuthorisation(listOf("ROLE_VIEW_CONTACTS")))
+        .exchange()
+        .expectStatus().isOk
+        .expectBody()
+        .jsonPath("length()").isEqualTo(2)
```

Comment:
This would previously have returned 3 addresses. due to the duplicate phones (2 phones at one address and 1 phone at the other)